python/ray/data/dataset.py

```diff
@@ -3628,6 +3630,45 @@ def _aggregate_result(self, result: Union[Tuple, TableRow]) -> U:
         else:
             return result
 
+    def _repr_html_(self) -> str:
```

Comment:
Nit: `num_blocks` and `num_rows` aren't really settings
 
<img width="260" alt="image" src="https://user-images.githubusercontent.com/26107013/182729956-a0eec02b-48a0-4570-997f-8d5cc5b3f375.png">

Reply:
Good point - what do you think of `Field` and `Value` as column names instead?

Reply:
That seems reasonable